packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -913,6 +914,12 @@ class SemanticsProperties extends DiagnosticableTree {
   /// on iOS and TalkBack on Android).
   final SemanticsSortKey? sortKey;
 
+  /// Adds additional information onto the semantics subtree.
+  ///
+  /// This property provides a way for rendering objects to annotate their
+  /// semantics children without directly access them.
+  final Set<SemanticsTag>? tagsForChildren;
```

Comment:
Can you add a test for the semantics widget to ensure that the tags are properly applied to the tree? (Currently we are only indirectly testing this via the RenderParagraph, but if we ever change how RenderParagraph works, this feature would be untested).